cmake/projects/Redis++/hunter.cmake

```
@@ -0,0 +1,26 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_report_broken_package)
+
+hunter_add_version(
+  PACKAGE_NAME
+  Redis++
+  VERSION
+  1.3.6
+  URL
+  "https://github.com/sewenew/redis-plus-plus/archive/refs/tags/1.3.6.tar.gz"
+  SHA1
+  650a9fc65c958119f5360ae7dc2341487a29ac6d
+)
+
+hunter_pick_scheme(DEFAULT url_sha1_cmake)
+
+hunter_cmake_args(Redis++ CMAKE_ARGS REDIS_PLUS_PLUS_BUILD_ASYNC=libuv REDIS_PLUS_PLUS_BUILD_TEST=OFF CMAKE_PREFIX_PATH=${HUNTER_INSTALL_PREFIX})
```

use multiple lines instead of a very long one for better git diffs in the future

and why is the `CMAKE_PREFIX_PATH=${HUNTER_INSTALL_PREFIX})` needed? This should be done automatically by hunter

```suggestion
hunter_cmake_args(Redis++ CMAKE_ARGS
  REDIS_PLUS_PLUS_BUILD_ASYNC=libuv
  REDIS_PLUS_PLUS_BUILD_TEST=OFF
)
```

---

> and why is the `CMAKE_PREFIX_PATH=${HUNTER_INSTALL_PREFIX})` needed? This should be done automatically by hunter

this is needed so redis is able to find libuv and hiredis packages.
this is the way they describe in there docs.

---

link to documentation please

and please try it without. I thought this path is set by Hunter itself and you should not be required to set it as argument